README.md

```diff
@@ -1,12 +1,31 @@
-# IVPT
-Interactive &amp; visual periodic table
+EN
+==
+
+BASIC
+--
+![image](https://user-images.githubusercontent.com/79650307/229216014-198ab642-c9e5-4fab-9b9c-a70afcfbe5b8.png)
+
+<br>
+Interactive &amp; visual periodic table written on PyQT5.
 
 This graphical app is to help people studying chemistry with calculations.
 
-Please, be ready that it't take up to a minute to launch at the first time.
-It's going to be faster if you have SSD instead of HDD.
+`Please, be ready that it't take up to a minute to launch at the first time.
+It's going to be faster if you have SSD instead of HDD.`
+
+<br>
+
+HOWTO USE
+--
+
+1. Okay, you are sitting on this one on your favorite subject - chemistry, and, let's say, the teacher asks you for the mass of Palladium and you just open the program, click on the element and you're done!
+
+![image](https://user-images.githubusercontent.com/79650307/229216159-484896e2-141a-4149-8041-f92f8c643c75.png)
+
+
+2. The teacher asks you the mass of nickel? No problem - click on the `Stoichiometry` button, enter an element or formula and press `eval`
```

Comment:
it's better to use passive voice here. it would sound less informal (the vast majority of github users are adults, who are no more studying at school)